UnityProject/Assets/Scripts/Core/Chat/Chat.cs

```diff
@@ -60,11 +65,28 @@ public static void InvokeChatEvent(ChatEvent chatEvent)
 				chatEvent.channels = channel;
 				if (chatEvent.originator.TryGetComponent<PlayerScript>(out var playerScript))
 				{
+					//There are some cases where the player might not have a dynamic item storage (like the AI)
+					if (playerScript.DynamicItemStorage == null)
+					{
+						Physics2D.OverlapCircleNonAlloc(playerScript.mind.body.AssumedWorldPos.To2(), searchRadiusForSphereResult, nonAllocPhysicsSphereResult);
```

Comment:
The Ai's body is the camera not the core so this wont work correctly.

playerScript.PlayerChatLocation.AssumedWorldPos();

Reply:
good catch, working on it as well as the test fail soon.

Reply:
Done